kbs/src/token/coco.rs

```diff
@@ -90,8 +94,8 @@ impl AttestationTokenVerifier for CoCoAttestationTokenVerifier {
             }
         }
 
-        let Some(trusted_store) = &self.trusted_certs else {
-            log::warn!("No Trusted Certificate in Config, skip verification of JWK cert of Attestation Token");
+        if self.trusted_certs.all_certificates().is_empty() {
+            warn!("No Trusted Certificate in Config, skip verification of JWK cert of Attestation Token");
```

Comment:
No verification on the token against trust anchor is dangerous. I could get the point that for debug case. We'd better add a configuration in config file to control this and by default the trust anchor verification is enforced.

Reply:
I'm not changing the logic. That's the same comment I was making in the cover letter.